Cargo.toml

```diff
@@ -30,6 +30,12 @@ strip = "symbols"
 opt-level = "s"
 lto = true
 
+[profile.release.package.cairo-rs]
+opt-level = 3
+
+[profile.release.package.plotters-cairo]
+opt-level = 3
+
```

Comment:
Is it worth adding `plotters` itself to have opt-level 3 as well?

Reply:
Wildcard with plotters and Cairo might do better here.

Reply:
Wildcard pattern wasn't supported sadly